.evergreen.yaml

```diff
@@ -70,31 +75,100 @@ functions:
           # Get all artifacts and turn them into a list with --attach flags
           export ARTIFACTS_ATTACH=$(ls artifacts | xargs -I {} echo "--attach artifacts/{} ")
 
-          hub release create --message "Atlas Service Broker ${version}" $ARTIFACTS_ATTACH v${version}
+          ${workingdir}/bin/hub release create --message "Atlas Service Broker ${version}" $ARTIFACTS_ATTACH v${version}
   "setup_docker":
     - command: shell.exec
       params:
         script: |
           sudo pacman --refresh --noconfirm -S community/docker
           systemctl start docker.service
+  "teardown_docker":
+    - command: shell.exec
+      params:
+        script: |
+          # Remove all Docker images
+          docker rmi -f $(docker images -a -q) &> /dev/null || true
   "build_docker":
     - command: shell.exec
       params:
         working_dir: src/atlas-service-broker
         script: |
-          docker build . -t ${docker_repo}/${docker_name}:${version}
+          docker build . -t ${docker_repo}/${docker_name}
   "publish_docker":
     - command: shell.exec
       params:
         working_dir: src/atlas-service-broker
         script: |
           docker login --username ${docker_username} --password ${docker_password} ${docker_repo}
 
+          docker image tag ${docker_repo}/${docker_name} ${docker_repo}/${docker_name}:${version}
           docker push ${docker_repo}/${docker_name}:${version}
+
+          # Tag image as "latest" if version is stable.
           if [[ ${version} =~ [0-9]+.[0-9]+.[0-9]+$ ]]; then
             docker image tag ${docker_repo}/${docker_name}:${version} ${docker_repo}/${docker_name}:latest
             docker push ${docker_repo}/${docker_name}:latest
           fi
+  "setup_kubernetes":
+    - command: shell.exec
+      params:
+        script: |
+          export PATH="${workdir}/bin:$PATH"
+
+          echo "Install kubectl"
+          curl -L -o bin/kubectl https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl
+          chmod +x bin/kubectl
+
+          echo "Install kind"
+          curl -L -o bin/kind https://github.com/kubernetes-sigs/kind/releases/download/v0.4.0/kind-linux-amd64
+          chmod +x bin/kind
+
+          echo "Start cluster"
+          kind create cluster 
+          export KUBECONFIG="$(kind get kubeconfig-path)"
```

Comment:
So this is not applicable in the cases where we are using Evergreen expansions? E.g. `export ATLAS_BASE_URL=${atlas_base_url`

Reply:
Simple variable reassignment doesn't matter because there is no resultant error code. It only matters if the value on the right hand side of the assignment could fail.

Reply:
I see! Fixed